alphadia/outputtransform.py

```diff
@@ -365,7 +365,11 @@ def build(
         )
         _ = self.build_stat_df(folder_list, psm_df=psm_df, save=True)
         _ = self.build_lfq_tables(folder_list, psm_df=psm_df, save=True)
-        _ = self.build_library(base_spec_lib, psm_df=psm_df, save=True)
+        _ = self.build_library(
+            base_spec_lib,
+            psm_df=psm_df,
+            save=self.config["general"]["save_mbr_library"],
+        )
 
         if self.config["transfer_library"]["enabled"]:
             _ = self.build_transfer_library(folder_list, save=True)
```

Comment:
should we make saving those guys also optional?

Reply:
Are they not already optional and only performed if transfer library enabled =True?

Reply:
I was thinking  about the saving only ..